runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java

```diff
@@ -87,4 +87,14 @@ default int fakeXpDropDelay()
 		return 0;
 	}
 
+	@ConfigItem(
+		keyName = "disableFakeXpDrops",
+		name = "Disable fake xp drops",
+		description = "Disables post-200m fake exp drops",
```

Comment:
Mentioned this before in the other comment thread, this also applies to ironman XP drops gained from attacking other players in the wilderness.